network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/tck/MainConnectedComponentWithSwitchTest.java

```diff
@@ -15,6 +15,6 @@ public class MainConnectedComponentWithSwitchTest extends AbstractMainConnectedC
 
     @Override
     public void test() {
```

Comment:
This overrided test method could be deleted if you synchronize bus angle value too ...
See above